flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultSubpartitionView.java

```diff
@@ -50,6 +50,10 @@
 
 	boolean isReleased();
 
+	boolean isBlockedByCheckpoint();
```

Comment:
As above commented, if we can avoid task thread touching the blocked state, then together with https://github.com/apache/flink/pull/11351/files#r402899871, we can remove this method completely.

Reply:
Actually i want this state already be covered by part of existing `isAvailable` method.